docs/pages/includes/install-linux.mdx

```diff
@@ -5,8 +5,14 @@
         # Download Teleport's PGP public key
         $ sudo curl https://deb.releases.teleport.dev/teleport-pubkey.asc \
           -o /usr/share/keyrings/teleport-archive-keyring.asc
-        # Add the Teleport APT repository
-        $ echo "deb [signed-by=/usr/share/keyrings/teleport-archive-keyring.asc] https://deb.releases.teleport.dev/ stable main" \
```

Comment:
I wonder if we should keep the older repos in a separate tab like "Debian/Ubuntu (Legacy)" or something like this, along with a note explaining that these are the old repos, benefits of the new ones and saying the old ones will be supported for foreseeable future.

@fheinecke @ptgott Thoughts?

Reply:
The "Debian/Ubuntu Legacy (DEB)" tab in the current version works for me!